src/app/clients/getClientParcelsData.ts

```diff
@@ -0,0 +1,60 @@
+import supabase from "@/supabaseClient";
+import { DatabaseError } from "@/app/errorClasses";
+import { Data } from "@/components/DataViewer/DataViewer";
+import { formatDatetimeAsDate } from "@/app/parcels/getExpandedParcelDetails";
+
+export type RawClientParcelsDetails = Awaited<ReturnType<typeof getRawClientParcelsDetails>>;
+type ClientParcelDetails = RawClientParcelsDetails[number];
+export interface ParcelsDetail {
+    parcel_id: string;
+    collection_centre?: { name: string } | null;
+    packing_datetime: string | null;
+    voucher_number?: string | null;
+}
+export const getClientParcelsDetails = async (
+    clientId: string
+): Promise<ExpandedClientParcelDetails[]> => {
+    const rawClientParcelsDetails = await getRawClientParcelsDetails(clientId);
+    const formattedList = rawClientParcelsDetails.map(rawDataToClientParcelsDetails);
+    return formattedList;
+};
+const getRawClientParcelsDetails = async (clientId: string): Promise<ParcelsDetail[]> => {
```

Comment:
I'd suggest adding an empty line between each of those types / consts to provide separation